<launch>

  <!-- when running comment the absolute path at thermal.cpp -->

  <!-- =========================== The namespace =========================== -->
  <arg name="namespace" default="hole_detector" />

    <arg name="path" default="/home/aggelos/Desktop/flir_rosbag/bag4.bag" />

    <node pkg="rosbag" type="play" name="player" output="screen" args="--clock -l $(arg path)" >
    <remap from="/camera/depth_registered/points" to="/kinect/depth_registered/points" />
    </node>

  <arg name="output" default="screen" />


  <node
    name="thermal_camera_node"
    type="thermal_camera_node"
    pkg="pandora_vision_hole"
    respawn="False"
    output="$(arg output)"
    ns="$(arg namespace)">

    <rosparam file="$(find pandora_vision_hole)/cfg/thermal_node_standalone_topics.yaml"
    command="load" />

    <rosparam file="$(find pandora_vision_hole)/cfg/thermal_node_matching_variables.yaml"
    command="load" />    
  </node>

</launch>
